tests/test_docs_autosummary/index.rst

```diff
@@ -0,0 +1,8 @@
+.. autosummary::
+    :toctree: generated/
+    :template: matmodule.rst
```

Comment:
Is there documentation somewhere on when matmodule.rst, matclass.rst, or matbase.rst should be used?

Reply:
matmodule.rst, matclass.rst, matbase.rst are used as default template file for modules classes and others. It could be omitted. And users could create their own template files.

I add a simple explaination in the file.